sdk/core/azure-core/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 1.15.1 (Unreleased)
 
+### Breaking Changes
+
+- Sync stream downloading now raises `azure.core.exceptions.DecodeError` rather than `requests.exceptions.ContentDecodingError`
```

Comment:
Are we okay with this breaking change because no one should have written code to explicitly handle this error?

Reply:
Yes. Checked with @johanste and he agreed this was an acceptable breaking change.